src/parameterizations/lateral/MOM_hor_visc.F90

```diff
@@ -1907,6 +2084,10 @@ subroutine hor_visc_init(Time, G, GV, US, param_file, diag, CS, ADp)
                  "If true, use a biharmonic Leith nonlinear eddy "//&
                  "viscosity.", default=.false., do_not_log=.not.CS%biharmonic)
   if (.not.CS%biharmonic) CS%Leith_Ah = .false.
+  call get_param(param_file, mdl, "USE_LEITHY", CS%use_Leithy, &
+                 "If true, use a biharmonic Leith nonlinear eddy "//&
+                 "viscosity together with a harmonic backscatter.", &
+                 default=.false., do_not_log=.not.(CS%biharmonic .and. CS%Laplacian))
```

Comment:
I think `USE_LEITHY` should always be logged and we should add something like "Both BIHARMONIC and LAPLACIAN must be enabled when using this scheme". Otherwise, users will not know if this scheme is available unless `BIHARMONIC = True` and `LAPLACIAN = True`. There is already a check with a FATAL_ERROR to make sure that both of these flags are true when USE_LEITHY is enabled.

Reply:
Good point. I find myself checking the parameter file to see what is available, so it would be good to have it logged there.